### ISSUE TITLE
ColumnMaps cleared when importing an object with a "Type" property

### ISSUE THREAD

**Issue by simonlary** [eXpandFramework/eXpand#149]
The ColumnMaps of an ExcelImport is cleared after an object with a property named `Type` is imported. There are missing parenthesis in the if condition of the `ObjectSpaceOnObjectChanged` in `ExcelImportDetailViewController`. Right now it is : 
```
if (Equals(e.Object, ExcelImport) && e.PropertyName == nameof(BusinessObjects.ExcelImport.FullName) ||
                e.PropertyName == nameof(BusinessObjects.ExcelImport.SheetName) ||
                e.PropertyName == nameof(BusinessObjects.ExcelImport.UseHeaderRows)||
                e.PropertyName == nameof(BusinessObjects.ExcelImport.Type))
```
It should be :
```
if (Equals(e.Object, ExcelImport) && (e.PropertyName == nameof(BusinessObjects.ExcelImport.FullName) ||
                e.PropertyName == nameof(BusinessObjects.ExcelImport.SheetName) ||
                e.PropertyName == nameof(BusinessObjects.ExcelImport.UseHeaderRows)||
                e.PropertyName == nameof(BusinessObjects.ExcelImport.Type)))
```

**Comment by apobekiaris** [comment]
Thanks for the fix, I patched it in next minor 18.1.5.5. Let me know please, if someone needs an installation before then.

**Comment by apobekiaris** [comment]
Closing issue for age. Feel free to reopen it at any time.

.Thank you for your contribution.
